Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -3476,10 +3476,10 @@
     "NotSupportedByDesign": "javascript",
     "Results": [
       {
-        "Text": "año 247",
+        "Text": "el año 247",
```

Comment:
Prefereably we shouldn't tag the article, if possible. It could be reasonable to keep it, if altering this behaviour would be too problematic.

Reply:
Ok removed. 
But then a problem occurs (for both English and Spanish) when the year is written in letters e.g. "in the year two thousand twelve". In such a case, two separate results are produced "the year" (matched by OneWordPeriodRegex) and "two thousand twelve"